packages/expo-modules-autolinking/scripts/ios/xcode_env_generator.rb

```diff
@@ -1,6 +1,24 @@
 require 'open3'
 require 'pathname'
 
+def generate_or_remove_xcode_env_updates_file!()
+  project_directory = Pod::Config.instance.project_root
+  xcode_env_file = File.join(project_directory, '.xcode.env.updates')
+
+  ex_updates_native_debug = ENV['EX_UPDATES_NATIVE_DEBUG'] == '1'
+  if ex_updates_native_debug
+    Pod::UI.info "EX_UPDATES_NATIVE_DEBUG is set; auto-generating `.xcode.env.updates` to disable packager and generate debug bundle"
+    if File.exist?(xcode_env_file)
+      File.delete(xcode_env_file)
+    end
+    File.write(xcode_env_file, "export FORCE_BUNDLING=1\nunset SKIP_BUNDLING\nexport RCT_NO_LAUNCH_PACKAGER=1\n")
```

Comment:
Consider a squiggly [heredoc](https://ruby-doc.org/core-2.5.0/doc/syntax/literals_rdoc.html#:~:text=can%20use%20a-,%E2%80%9Csquiggly%E2%80%9D%20heredoc,-%2C%20which%20uses%20a):

```suggestion
    File.write(xcode_env_file, <<~EOS
      FORCE_BUNDLING=1
      unset SKIP_BUNDLING
      export RCT_NO_LAUNCH_PACKAGER=1
EOS)
```